util/kvcache/simple_lru.go

```diff
@@ -35,6 +36,20 @@ type cacheEntry struct {
 	value Value
 }
 
+var (
+	// GlobalLRUMemUsageTracker tracks all the memory usage of SimpleLRUCache
+	GlobalLRUMemUsageTracker *memory.Tracker
```

Comment:
Better to add a test case covering `SimpleLRUCache.Put` interface and check `GlobalLRUMemUsageTracker` tracks reasonable memory it consumes.

Reply:
update